mautic/mautic
# Fix yes/no button for emails

closes https://github.com/mautic/mautic/pull/10259
closes #10231 

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #10231 




#### Description:
Switch active/not active email not work on emails list



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to emails
3. Try switch off/on
4. See the error
5. After PR should work properly
